src/math/p5.Vector.js

```diff
@@ -2207,19 +2220,18 @@ p5.Vector.dist = function dist(v1, v2) {
  * @param {p5.Vector} v1
  * @param {p5.Vector} v2
  * @param {Number} amt
- * @param {p5.Vector} target if undefined a new vector will be created
- */
-/**
- * @method lerp
- * @static
- * @param {p5.Vector} v1
- * @param {p5.Vector} v2
- * @param {Number} amt
- * @return {Number}      the lerped value
```

Comment:
why was this removed?

Reply:
@lmccart I remove all of these to make target value [target] (optional) as the functions are always returning resultant vector

Reply:
Should I revert these changes?